documentation/src/main/asciidoc/topics/con_indexing_annotations.adoc

```diff
@@ -19,7 +19,7 @@ There are two annotations that control the entities and fields that {brandname}
 
 |`store`
 |Allows {brandname} to store fields in indexes so you can use them for projections.
-|`Store.YES` or `Store.NO`
+|`Store.YES` or `Store.NO`. Use `Store.YES` and set `sortable = true` for fields that need to be used for sorting.
```

Comment:
The store options does not exist anymore.
The new way to mark a field sortable is to set `sortable = true`

Reply:
Hi @fax4ever thank you for checking the PR. I guess you're speaking about Infinispan 15? This is a clarification for Infinispan 13 and previous versions so the old way.